librz/core/ctypes.c

```diff
@@ -306,19 +306,19 @@ static void core_types_struct_print(RzCore *core, RzBaseType *btype, RzOutputMod
 	case RZ_OUTPUT_MODE_JSON: {
 		rz_return_if_fail(pj);
 		pj_o(pj);
-		if (btype && !rz_vector_empty(&btype->struct_data.members)) {
-			pj_ks(pj, "name", btype->name);
-			pj_k(pj, "members");
-			pj_o(pj);
+		pj_ks(pj, "name", btype->name);
+		pj_k(pj, "members");
+		pj_o(pj);
+		if (!rz_vector_empty(&btype->struct_data.members)) {
```

Comment:
this is probably unnecessary. rz_vector_foreach will not do anything if the vector is empty .